topo/node/arista/arista.go

```diff
@@ -366,10 +366,21 @@ func defaults(pb *tpb.Node) *tpb.Node {
 		pb.Labels["vendor"] = tpb.Vendor_ARISTA.String()
 	}
 	if pb.Labels["model"] == "" {
-		pb.Labels["model"] = pb.Model
+		if pb.Labels["model"] = pb.Model; pb.Model == "" {
+			pb.Model = "ceos"
+			pb.Labels["model"] = pb.Model
+		}
+	} else {
+		pb.Model = pb.Labels["model"]
 	}
+
 	if pb.Labels["os"] == "" {
-		pb.Labels["os"] = pb.Os
+		if pb.Labels["os"] = pb.Os; pb.Os == "" {
```

Comment:
this can be simplified by defaulting pb.Os and pb.Model above this block. then labels can remain as is

Reply:
Fixed.